src/views/partials/blogs/create.hbs

```diff
@@ -0,0 +1,20 @@
+<form action="/blogs/create" method="post" autocomplete="off" enctype="application/json">
+  <input name="id" type="hidden" value=" ">
+  <div class="form-group">
+    <label for="title">Title:</label>
+    <input type="text" class="form-control" id="title" name="title" style="width:700px" required>
+  </div>
+  <input id="today" name="date" type="hidden" value=" ">
+  <div class="form-group">
+    <label for="author">Author:</label>
+    <input type="text" class="form-control" id="title" name="author" style="width:700px" required>
+  </div>
+  <div class="form-group">
+    <label for="content">Content:</label>
+    <textarea name="content" class="form-control" id="content" style="width:700px; height:300px;">
```

Comment:
Can you please not use inline-styles but CSS instead? :)

Reply:
Sure :)